source/PlayerInfo.cpp

```diff
@@ -465,13 +465,16 @@ void PlayerInfo::Save() const
 		if(saved.GetDate() != date.ToString())
 		{
 			string root = filePath.substr(0, filePath.length() - 4);
-			string files[4] = {
+			static const int SAVE_COUNT = 6;
+			string files[SAVE_COUNT] = {
+				root + "~~previous-5.txt",
+				root + "~~previous-4.txt",
 				root + "~~previous-3.txt",
 				root + "~~previous-2.txt",
 				root + "~~previous-1.txt",
```

Comment:
cant we do this automatically, depending on the savecount, by appending to it?
that way we could eventually define the save count in a more flexible way (or just change it when we feel like it)